### PR TITLE
updated cartfile to point RxSwift to 4.3.1

### PR DESCRIPTION
**Pull request by raviranderia**

### CODE REVIEW

File: Cartfile [hunk 1]
```diff
@@ -1 +1 @@
-github "ReactiveX/RxSwift" ~> 4.0
+github "ReactiveX/RxSwift" ~> 4.3.1
```
Comment (bartoszstelmaszuk) [inline Cartfile]: Basically `~> 4.0` will do almost the same as `~> 4.3.1`. It means "install RxSwift with version at least 4.0 but lower than 5.0". What it needs to be done is to run `Cartfile update` to get new version of RxSwift to library (see `Cartfile.resolved`)

Comment (raviranderia) [reply Cartfile]: I did run carthage, but it was still always getting 4.0 for some reason.

Comment (bartoszstelmaszuk) [reply Cartfile]: did you run firstly `carthage upgrade`?

Comment (raviranderia) [reply Cartfile]: yes

**Pull request closed without merging**